### PR TITLE
Add creating graph and uploading models in action

### PR DESCRIPTION
**Pull request by chalenge**
## Description

Add function to create graph and uploading cicd models in action when posting the build and deployment infor

## Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## Related issues

Related to https://github.com/faros-ai/tickets/issues/189

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -64,3 +65,13 @@ jobs:
           build-source: ${{ steps.emit-build-info.outputs.org-source }}
           started-at: ${{ steps.set_start_time.outputs.started-at }}
           ended-at: ${{ steps.set_end_time.outputs.ended-at }}
+
+      - name: Delete test graph
```
Comment (cjwooo) [inline .github/workflows/test.yml:69]: Wouldn't this create issues if there's multiple concurrent GHA test workflows running, and one workflow deletes a graph while the others are still running? This repo isn't updated often so maybe it's not worth working around it.

Comment (chalenge) [reply .github/workflows/test.yml:69]: Yep, that's right. Will remove. Didn't want to keep the graph around but maybe fine

File: dist/resources/models/cicd-vcs.gql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# This model was extracted from the cicd schema to remove that repo's
```
Comment (tovbinm) [inline dist/resources/models/cicd-vcs.gql:1]: why do we have a copy of our models here? @chalenge

File: package.json [hunk 3]
```diff
@@ -6,10 +6,12 @@
   "main": "lib/main.js",
   "scripts": {
     "build": "tsc",
+    "clean": "rimraf dist/",
+    "copy-models": "copyfiles -u 2 \"node_modules/faros-canonical-models/resources/**/*.gql\" dist/",
     "format": "prettier --write **/*.ts",
     "format-check": "prettier --check **/*.ts",
     "lint": "eslint src/**/*.ts",
-    "package": "ncc build --source-map --license licenses.txt",
+    "package": "ncc build --source-map --license licenses.txt && npm run copy-models ",
```
Comment (tovbinm) [inline package.json:14]: why are we doing this hack? this would be very difficult to maintain.

**Pull request closed without merging**